content/post/articles/typewriter-animation-with-css.md

```diff
@@ -0,0 +1,184 @@
+---
+title: The Typewriter Animation
+date: 2022-01-27T00:15:28+01:00
+images: []
+categories: [tutorial, blogging, newbies, fun]
+tags: [typewriter, animation, css, javascript]
+authors: ["Yokwejuste"]
+---
+<style>
+    img{
+        border-radius: 10px;
+    }
+    .writer {
+        font-family: Courier, monospace;
+        display: inline-block;
+        }
+        .writer-text {
+            display: inline-block;
+            overflow: hidden;
+            letter-spacing: 2px;
+            animation: typing 5s steps(30, end), blink .75s step-end infinite;
+            white-space: nowrap;
+            font-size: 30px;
+            font-weight: 700;
+            border-right: 4px solid orange;
+            box-sizing: border-box;
+        }
+
+        @keyframes typing {
+            from { 
+                width: 0%
+            }
+            to { 
+                width: 100%
+            }
+        }
+
+        @keyframes blink {
+            from, to { 
+                border-color: transparent 
+            }
+            50% { 
+                border-color: orange; 
+            }
+        }
+</style>
+## The ultimate solution to typewriter animation 💪
+
+![typewriter](https://i0.wp.com/css-tricks.com/wp-content/uploads/2021/07/typewriter-animation-text.gif?fit=900%2C450&ssl=1)
+Hey buddy, for the web newbies I've something fun today.
+Let's talk about something amazing, the Typewriter animation. 
+
+It looks like this: 👇
+<div class="writer">
+    <div class="writer-text">
+    Happy to contribute for OSS-Cameroon blog.
+    </div>
+</div>
+
+After long research, I found that it could be summarised into 4 steps:
+
+### Step 1: Write the HTML code
+Write the HTML code to display the text.
+```
+<div class="writer">
+    <div class="writer-text">Happy to contribute for OSS-Cameroon blog</div>
+</div>
+```
+### Step 2: Choosing an appropriate font
+choose a monospace font type, like "Courier New", something sans-serif too. This is to simply ensure that the animation property takes one character at a time since monospace fonts are always of equal width.
```

Comment:
```suggestion
choose a monospace font type, like "Courier New", something sans-serif too. This is to simply ensure that the animation properly takes one character at a time since monospace fonts are always of equal width.
```

Reply:
Nothing to change on that part! Working on a gift to put there.